staging/src/k8s.io/client-go/tools/cache/controller_test.go

```diff
@@ -295,7 +299,18 @@ func TestHammerController(t *testing.T) {
 
 	// Let's wait for the controller to finish processing the things we just added.
 	// TODO: look in the queue to see how many items need to be processed.
-	time.Sleep(100 * time.Millisecond)
+	wait.PollWithContext(ctx, 100*time.Millisecond, wait.ForeverTestTimeout, func(ctx context.Context) (bool, error) {
```

Comment:
wait.PollWithContext is Deprecated:
https://github.com/kubernetes/apimachinery/blob/v0.32.1/pkg/util/wait/poll.go#L81

I believe `wait.PollUntilContextTimeout` should be used instead now

Reply:
Thanks! Updated to use `wait.PollUntilContextTimeout`.